.github/workflows/e2e.yml

```diff
@@ -31,6 +31,7 @@ jobs:
       - name: Install requirements.txt
         run: |
           pip install -r requirements/github.txt
+          pip install umap-learn
```

Comment:
Can we instead make `requirements/e2e.txt` and have it look like the following, and change it here?

```
-r github.txt

umap>=0.5.3
```

Reply:
buen idea!